src/runtime.js

```diff
@@ -465,7 +464,7 @@ export class MultidocManager {
      * @param {!VisibilityState} state
      */
     amp['setVisibilityState'] = function(state) {
```

Comment:
just curious: is it a public API for PWA? or is it just for debugging?

Reply:
Yes. Public API. A shell application can manage document visibility similar to viewer.